### DISCUSSION TITLE
Web development

### DISCUSSION THREAD

**Discussion by TanyaSthole** [codewars/codewars.com#3011] [General]
Hello guys i am a high school student interested in web development and i would like help on the material i need or s
hould use

**Comment by Galadien** [comment]
[The Odin Project](https://www.theodinproject.com/) and [FreeCodeCamp](https://www.freecodecamp.org/) are great places to start (I use both) and are free.

**Comment by hobovsky** [comment]
Github discussions are meant to discuss ideas and functions related to Codewars platform. For help with learning please visit Codewars Discord and its topic-related channels or `#posts`.
